### DISCUSSION TITLE
Spotify detects "suspicious activity" on my account

### DISCUSSION THREAD

**Discussion by ghost** [KRTirtho/spotube#1759] [Q&A]
A few minutes after logging in, I got an e-mail from Spotify saying that they reset my password due to suspicious activity. Is there a way to circumvent this?

**Comment by KRTirtho** [comment]
It's random. If you're login/logout too much in a day or a certain period of time they send this email. Sometimes it's sent for only logging once. Best option will be turning on 2 factor authentication
